xcode/Subconscious/Shared/Components/AppView.swift

```diff
@@ -1369,14 +1457,24 @@ struct AppModel: ModelProtocol {
         environment: AppEnvironment
     ) -> Update<AppModel> {
         do {
-            let sphereIdentity = try state.sphereIdentity.unwrap()
-            let did = try Did(sphereIdentity).unwrap()
-            let sphereVersion = try environment.database.readSphereSyncInfo(
-                sphereIdentity: did
-            ).unwrap()
-            logger.log("Database last-known sphere state: \(sphereIdentity) @ \(sphereVersion)")
+            let info: OurSphereRecord = try environment.database
+                .readOurSphere()
+                .unwrap()
+            logger.log(
+                "Last index for our sphere",
+                metadata: [
+                    "identity": info.identity.description,
+                    "version": info.since
+                ]
+            )
         } catch {
-            logger.log("Database last-known sphere state: unknown")
+            logger.log(
+                "Last index for our sphere",
+                metadata: [
+                    "identity": "nil",
+                    "version": "nil"
```

Comment:
My rationale at the time was that logs end up being strings, and it might be better to leave it to the caller to decide how to serialize values (incl optionals) to strings.